lib/commands/commit.js

```diff
@@ -48,12 +48,18 @@ module.exports = {
       return runCommand('git add . && git commit -m "' + options.message + '"', execOptions)
     }
 
+    function returnToPreviousCheckout() {
+      // Each '\' needed to be escaped
+      return runCommand("git checkout `git reflog HEAD | sed -n '/checkout/ !d; s/.* \\(\\S*\\)$/\\1/;p' | sed '2 !d'`", execOptions);
```

Comment:
The bash command is longer than 80 chars. 

I tried to wrap it but it just obfuscates the sub command which is hard enough to read. 

I think this line should be an exception...